src/lib/logical_query_plan/jit_aware_lqp_translator.hpp

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 #include "lqp_translator.hpp"
-#include "operators/jit_operator/operators/jit_compute.hpp"
```

Comment:
In file `jit_aware_lqp_translator.cpp` the class `JitCompute` is used. So if the include is removed here, I would suggest to add it to to the file `jit_aware_lqp_translator.cpp`. Otherwise compilation is failing at least on my mac.